install/gcp/installer/configs/server_config.py

```diff
@@ -37,6 +39,7 @@ def __init__(self, **kwargs):
         self.notification_recipient_email = (
             kwargs.get('notification_recipient_email'))
         self.gsuite_superadmin_email = kwargs.get('gsuite_superadmin_email')
+        self.skip_sendgrid_config = bool(kwargs.get('skip_sendgrid_config'))
```

Comment:
If I am not mistaken, action='store_true' will store the boolean value (True/False), is the cast here necessary?

Reply:
I did it based on this: 
https://github.com/GoogleCloudPlatform/forseti-security/blob/2.0-rc3/install/gcp/installer/configs/config.py#L36

Reply:
ok, let's stick with this style for now then